tidb-configuration-file.md

```diff
@@ -212,12 +219,13 @@ Configuration items related to log.
 - Outputs the threshold value of consumed time in the slow log.
 - Default value: `300ms`
 - If the value in a query is larger than the default value, it is a slow query and is output to the slow log.
+- Since v6.1.0, the threshold value of consumed time in the slow log is determined by configuration [`instance.tidb_slow_log_threshold`](/tidb-configuration-file.md#tidb_slow_log_threshold) or system variable [`tidb_slow_log_threshold`](/system-variables.md#tidb_slow_log_threshold). `slow-threshold` still takes effect. But if `slow-threshold` and `instance.tidb_slow_log_threshold` are setting at the same time, TiDB will apply the value of `instance.tidb_slow_log_threshold`.
```

Comment:
```suggestion
- Since v6.1.0, the threshold value of consumed time in the slow log is specified by the TiDB configuration item [`instance.tidb_slow_log_threshold`](/tidb-configuration-file.md#tidb_slow_log_threshold) or the system variable [`tidb_slow_log_threshold`](/system-variables.md#tidb_slow_log_threshold). `slow-threshold` still takes effect. But if `slow-threshold` and `instance.tidb_slow_log_threshold` are set at the same time, TiDB will apply the value of `instance.tidb_slow_log_threshold`.
```